core/tests/local_cluster.rs

```diff
@@ -235,7 +235,7 @@ fn test_listener_startup() {
 }
 
 #[test]
-#[serial]
+#[ignore]
```

Comment:
I think you can do away with this after https://github.com/solana-labs/solana/commit/fc180f4cbf4416580e8b207fa5e3fd4b250ef52c.  I haven't seen it fail since

Reply:
thanks.  this should not have been in the PR.